### PR TITLE
Add updates to clustering README

### PR DESCRIPTION
**Pull request by cansav09**
This PR addresses #54 and #48 to the clustering module.

### CODE REVIEW

**Comment by cansav09** [comment]
Okay, take a look at this one now that we are leaving the create_gct_file.R script file where it is for now

**Comment by jaclyn-taroni** [comment]
@cansav09 I would go through the recommendations in the review for #66 and carry that over here

**Comment by cansav09** [comment]
I applied the changes we agreed on for #52 to here, except without the CLS file instructions since that's not needed for the clustering GenePattern stuff. Let me know what you guys think. Most of this is repeated from other places.

File: clustering/README.md [hunk 1]
```diff
@@ -1,32 +1,148 @@
 # refine.bio Examples: Clustering
-This directory contains example workflows of how to filter, cluster and create
-a heatmap from data downloaded from [refine.bio](https://www.refine.bio).
 
-***
-### Clustering data in R
 [This notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/clustering/clustering_example.Rmd)
```
Comment (jaclyn-taroni) [inline clustering/README.md]: This has not been updated to the rendered HTML notebook

File: clustering/README.md [hunk 2]
```diff
@@ -1,32 +1,148 @@
...
+See [this guide using to R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html#using-notebooks) for more information about inserting and executing code chunks.
+
+#### Using your own data
+
+For the example in this module, the gene expression data and sample metadata are stored in a `data/` directory.
+If you'd like to adapt an example to include data you've obtained from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory and changing the filenames and paths in the notebook to match these files.
+We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebook if you move notebooks outside of the GitHub repository structure.
+
+[The clustering R Notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/clustering/clustering_example.nb.html) could be manipulated to sort by another summary statistic to create a heatmap, depending on your dataset and your research questions.
```
Comment (jaclyn-taroni) [inline clustering/README.md]: This is also the incorrect link

File: clustering/README.md [hunk 3]
```diff
@@ -1,32 +1,148 @@
...
-a heatmap from data downloaded from [refine.bio](https://www.refine.bio).
 
-***
-### Clustering data in R
 [This notebook](https://github.com/AlexsLemonade/refinebio-examples/blob/master/clustering/clustering_example.Rmd)
 uses the Bioconductor R package `ComplexHeatmap` for clustering and creating a
-heatmap.
+heatmap from data downloaded from [refine.bio](https://www.refine.bio).
 In this example, the gene expression matrix is subset to only include genes with high variance.
 The high variances set of genes is then used to create a heatmap.
```
Comment (jaclyn-taroni) [inline clustering/README.md]: **This set of genes** is then 

Perhaps?

**Pull request merged**